xbmc/addons/VFSEntry.cpp

```diff
@@ -25,7 +25,22 @@ CVFSAddonCache::~CVFSAddonCache()
 void CVFSAddonCache::Init()
 {
   CServiceBroker::GetAddonMgr().Events().Subscribe(this, &CVFSAddonCache::OnEvent);
-  Update();
+
+  // Load all during Kodi start available VFS addons
```

Comment:
`// Load all available VFS addons during Kodi start`

Reply:
Thanks for comments! Have changed the text about.